torch/nn/quantized/modules/dropout.py

```diff
@@ -0,0 +1,26 @@
+import torch
+import torch.nn.quantized.functional
+
+class Dropout(torch.nn.Dropout):
```

Comment:
Thanks for fixing this.  It would be valuable to add some context to why the logic of this class is not actually doing dropout, i.e. there is currently no need or implementation for actual quantized dropout, and this is a placeholder to enable models where fp32 tensors had dropout to work with quantized tensors.

Reply:
I'm also still not sure why not we just use `nn.Identity` instead of creating this module, but I don't feel super strongly

Reply:
I don't have any preference.  
@jerryzh168 any thoughts?